<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>company</class>
 <widget class="QWidget" name="company">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1361</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="view_orders">
   <property name="geometry">
    <rect>
     <x>1056</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="Notificationlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Notifications:</string>
   </property>
  </widget>
  <widget class="QLabel" name="notificationCount">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="viewNewOrdersTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>1311</width>
     <height>181</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="orders_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>1311</width>
     <height>251</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>Customer Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Order ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Destination</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Distance (KM)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Height (M)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Length (M)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Width (M)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Weight (KG)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Lorry Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Shipping Cost (Â£)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Commission</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Assigned Driver</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Delivery status</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="Notificationlabel_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Orders</string>
   </property>
  </widget>
  <widget class="QLabel" name="Notificationlabel_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pending Orders</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_orders_2">
   <property name="geometry">
    <rect>
     <x>926</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>View Orders</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sign_out">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>10</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="feedbackBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>231</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Feedback</string>
   </property>
   <widget class="QLabel" name="noOfReviewText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of reviews:</string>
    </property>
   </widget>
   <widget class="QLabel" name="avgRatingText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Average rating:</string>
    </property>
   </widget>
   <widget class="QLabel" name="noOfReviews">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="avgRating">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
